### ISSUE TITLE
String unescaping bug and a few questions

### ISSUE THREAD

**Issue by mrakgr** [nunit/docs#108]
Hello. I am investigating the [issue of why](https://github.com/fsprojects/FsUnit/issues/96#issuecomment-243082543) the message string passed into assert unescapes the newline characters and I thought to ask here first. I am also looking at the source, but it is not really obvious to me just by the brief scan that I did. I have some questions as well.

1) If a class has an IDisposable interface implemented, will that automatically get called after the test is over or is an explicit TearDown attribute necessary? I did not see that noted in the manual, so it might be worth adding.

2) If the tests are run in parallel, will each thread create its own instance of the class being tested or will that class be shared? Knowing that might be important. I had not seen that noted in the manual as well.

3) In the [Platform Attribute page](https://github.com/nunit/docs/wiki/Platform-Attribute) there is a slight omission. Specifically:

```
If a test or fixture with the Platform attribute does not satisfy the specified platform requirements it is skipped. The test does not affect the outcome of the run at all: it is not considered as ignored and is not even counted in the total number of tests. [Ed.: Check this.] In the gui, the tree node for the test remains gray and the status bar color is not affected.
```

Note the `Ed.: Check this.` that was made all the way back in January. I am guessing the author completely forgot about it by now.

**Comment by mrakgr** [comment]
I tried building NUnit manually, and I can see that the string prints just fine there both in F# (when I reference the manually built .dll from Interactive) and C# (when I convert the .NET 4.5 version to a console app.) So that probably means that either the NUnit package that comes with FsUnit is stale, or if the case is that FsUnit fetches the main one directly from Nuget, then the main one on Nuget is stale.

**Comment by mrakgr** [comment]
I've tried a fresh project with just Nunit from Nuget and I can verify that it is stale as the bug persists with just it.

**Comment by jnm2** [comment]
I think this was discussed in the nunit repo and fixed for the future 3.5 release: https://github.com/nunit/nunit/issues/1575

**Comment by jnm2** [comment]
@CharliePoole people keep opening issues in docs instead of nunit. A new issue template with a warning might help bring it to people's attention, what do you think?

**Comment by CharliePoole** [comment]
@jnm2 Good idea about the docs repo!

**Comment by SeanKilleen** [comment]
🧹 Cleanup -- I am closing this issue because it is 3+ years old and does not appear to have an actionable resolution for the NUnit docs project at this time.

If you feel this is closed in error, don't hesitate to ask me to reopen. Thanks!
